### PR TITLE
DEVOPS-5485 Add support for internal acr registry

### PR DESCRIPTION
**Pull request by michal-jasinski-piwikpro**

### CODE REVIEW

File: dtools/push/action.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ inputs:
     required: false
     default: ${{ github.ref_name }}
     description: Image tag, that should be pushed to the registry
+  include-registry:
+    required: true
+    default: acr
+    description: Name of the registry where image should be pushed (acr, internal_acr)
```
Comment (mrfijas) [inline dtools/push/action.yaml:17]: > List of registry ids [default: ecr, acr, docker_hub, internal_acr]

https://github.com/PiwikPRO/Developers-Tools/blob/1e13425f360691ff1e0c77e3ec296fd5bdd4764a/src/main.py#L332-L338

so I believe it should be like "Comma-separated list of registries where image will be pushed to" or something

Comment (michal-jasinski-piwikpro) [reply dtools/push/action.yaml:17]: Do you mean in the description field or do you suggest changing the default to list of registries?

Comment (mrfijas) [reply dtools/push/action.yaml]: I meant description. Reading this I'd think that I can specify only one

Comment (michal-jasinski-piwikpro) [reply dtools/push/action.yaml]: Changed

**Pull request merged**